etc/visual-studio/libopenthread-windows.vcxproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
yea, it seems there is/was a byte order mark <U+FEFF> that my editor must have munched/dropped....
And - the NEWLINE (or not) at the end of the file.

Ugh.